lib/mosaic-utils/src/main/java/org/eclipse/mosaic/lib/util/FileUtils.java

```diff
@@ -92,4 +93,46 @@ public static Collection<File> searchForFilesWithName(Collection<File> rootFiles
                 f.canRead() && (f.isDirectory() || (f.isFile() && f.getName().toLowerCase().equals(fileName.toLowerCase())))
         );
     }
+
+    /**
```

Comment:
Why that? Already part of follow-up issue??

Reply:
I found these methods in a utility class in scenario-convert. I decided to remove this additional utility class and moved the methods to `FileUtils` instead. However, indeed only scenario-convert currently uses those.